app/models/migration_upload_snapshot.rb

```diff
@@ -18,6 +18,13 @@ def mark_complete(s3_file)
     if index.present?
       files[index]["migrate_status"] = "complete"
       files[index].delete("migrate_error")
+
```

Comment:
At some point we may want to change this to utilize the last_response from s3_query_service, but this works, so lets go with it.